pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1015,12 +1015,12 @@ protected void handleSubscribe(final CommandSubscribe subscribe) {
                                                     "[{}][{}][{}] Failed to create consumer because exclusive consumer"
                                                             + " is already connected: {}",
                                                     remoteAddress, topicName, subscriptionName,
-                                                    exception.getCause().getMessage());
+                                                    exception.getCause().getMessage(), exception);
                                         }
                                     } else if (exception.getCause() instanceof BrokerServiceException) {
                                         log.warn("[{}][{}][{}] Failed to create consumer: consumerId={}, {}",
                                                  remoteAddress, topicName, subscriptionName,
-                                                 consumerId,  exception.getCause().getMessage());
+                                                 consumerId,  exception.getCause().getMessage(), exception);
```

Comment:
This was done on purpose to avoid stack traces for "known" or expected exceptions.